package.json

```diff
@@ -4,7 +4,8 @@
   "description": "feature/_issue number_-_short\\_description_",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint **/*.js"
```

Comment:
Good to see you getting more familiar with `package.json` and `npm` scripts. Why don't we look into removing the `test` script for now until we add something like `mocha` or `Jest`?